CONTRIBUTING.md

```diff
@@ -3,7 +3,7 @@
 ## Overview
 
 This documents explains the processes and practices recommended for
-contributing enhancements to the Prometheus charm.
+contributing enhancements to the Prometheus Charmed Operator.
```

Comment:
At first glance it seems that "charmed operator" should not be capitalized here.